examples/file-feed/complete/src/browser-app/DataStore.js

```diff
@@ -0,0 +1,104 @@
+ // Need to include this to make webpack happy
+import { Buffer } from 'buffer/' // eslint-disable-line no-unused-vars
+import EventEmitter from 'events'
+import spawnNode from '../util/spawn-node'
+import OrbitDB from 'orbit-db'
+import { readFileContents } from './read-file'
+
+let instance
+
+class DataStore extends EventEmitter {
+  constructor (options) {
+    super()
+    spawnNode(options, (err, node) => {
+      if (err) {
+        console.log(err)
+      }
+      this.ipfs = node
+      this.emit('ready')
+    })
```

Comment:
Getting issues from this

![image](https://cloud.githubusercontent.com/assets/1211152/22395392/19a46caa-e535-11e6-9472-b7bd688324fe.png)